pycbc/detector.py

```diff
@@ -322,3 +304,65 @@ def overhead_antenna_pattern(right_ascension, declination, polarization):
 def effective_distance(distance, inclination, f_plus, f_cross):
     return distance / np.sqrt( ( 1 + np.cos( inclination )**2 )**2 / 4 * f_plus**2 + np.cos( inclination )**2 * f_cross**2 )
 
+def dist_lisa_gbd(det, ref_time):
+    L_pos = LISA().get_pos(ref_time)
+    if isinstance(det, str):
+        det = Detector(det, ref_time).location
+        det = (det*units.m / units.AU).decompose()
+    if isinstance(det, np.ndarray) and det.shape[0] == 3:
+        det = det
+    dist = np.array([det[0] - L_pos[0],
+                     det[1] - L_pos[1],
+                     det[2] - L_pos[2]])
+    return dist
+
+
+"""     LISA class      """
+
+
+class LISA(object):
+    def __init__(self):
+        None
+
+    def get_pos(self, t_gps):
+        t_gps = Time(val=t_gps, format='gps', scale='utc').jyear
+
+        n = np.array(range(1, 4))
+        kappa, _lambda_ = 0, 0
+        alpha = 2. * np.pi * t_gps/1 + kappa
```

Comment:
This doesn't seem to be a gps time anymore, some perhaps use a better label. Also, is there not a fiducial start time for this that is a specific year? Should that not be accounted for in here.